.github/workflows/benchmark.yml

```diff
@@ -184,6 +184,10 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Add synthetic huge data
```

Comment:
@csh519 
huge file with useless AnalysisTarget requires too much memory. It happens in some repos (by VoC)

Reply:
Yup. Looks good.